lib/maplibre-compose/src/commonMain/kotlin/dev/sargunv/maplibrecompose/compose/CameraState.kt

```diff
@@ -54,6 +55,10 @@ public class CameraState internal constructor(firstPosition: CameraPosition) {
   public val moveReason: CameraMoveReason
     get() = moveReasonState.value
 
+  /** meters per dp at the target position */
+  public val metersPerDpAtTarget: Double
```

Comment:
Small suggestion: Rename to `metersPerDp`? The `CameraState` already represents, well, the current state of the camera, including `position`, so it should be clear from context what is meant.
(Also, "target position" is mentioned nowhere else)

Reply:
I wanted to clarify a bit because the meters per dp varies by longitude, pretty significantly if zoomed out. So this is the scale at the center of the view (if camera padding is zero). Or at the target of the view.

Happy to rename it though. we should clarify in the docs regardless of name

Reply:
Separately, I made the scale bar work like the MLN one, showing scale at the target, even though the bar is in the corner. 

But I think it's more intuitive for the bar to show scale exactly where it's at, as at least when I use a map I pan something I want to measure over to the scale bar to measure it. This is what the Google scale bar does, but I dropped that functionality for simplicity (and to test out this state).

Reply:
So there will likely be additional functions "meters per dp at longitude" and "meters per dp at screen location". Whatever naming we decide for this state should probably be consistent with those functions too